src/index.js

```diff
@@ -47,6 +47,7 @@ const initOnce = (fn, message) => (...args) => {
   window._fs_initialized = true;
 };
 
-wrappedFS.init = initOnce(init, 'FullStory init has already been called once. Additional invocations are ignored');
+wrappedFS.init = initOnce(init, 'FullStory init has already been called once, additional invocations are ignored');
+wrappedFS.anonymize = () => wrappedFS.identify(false);
 
```

Comment:
I added `anonymize` because:

1. There are plans to add `anonymize` to the snippet anyway
1. I twisted myself in knots trying to figure out a meaningful way to express `identify(false)` as a typed signature: `identify(dontAnonymize: boolean)` is both a double-negative (ew) and implies that this function call may be required because anonymization is default behavior